src/engine/ResultTable.h

```diff
@@ -49,8 +50,7 @@ class ResultTable {
    */
   vector<size_t> _sortedBy;
 
-  vector<vector<Id>> _varSizeData;
-  void* _fixedSizeData;
+  IdTable _data;
```

Comment:
I like `_data` (or maybe something like `_result`) because it does not imply anything about the type of the values, which may be Ids but could also be floats, integers or something entirely different (that fits into 64 bits).

Reply:
Ok yeah that makes sense, lets safe the work and keep it `_data`